config/locales/en.yml

```
@@ -281,6 +281,8 @@ en:
               blank: cannot be blank
             third_party_address:
               blank: cannot be blank
+            third_party_email:
+              invalid: enter an email address in the correct format, like name@example .com
```

There's an extra space that can be removed
```suggestion
              invalid: enter an email address in the correct format, like name@example.com
```